alter_configs_request.go

```diff
@@ -0,0 +1,146 @@
+package sarama
+
+type AlterConfigsRequest struct {
+	Resources    []*AlterConfigsResource
+	ValidateOnly bool
+}
+
+type AlterConfigsResource struct {
+	T             ResourceType
```

Comment:
I'd prefer a more useful name here... I guess `Type` is reserved and `ResourceType` is already the name of the struct, but maybe `ConfigResourceType` or something? Or make that the name of the struct and call the element just `ResourceType`?

Reply:
😊 I meant to do a second pass to rename this. I _think_ we can use `Type` here, since it's `type` that's reserved